hax-lib/src/lib.rs

```diff
@@ -15,6 +15,8 @@
 
 #![no_std]
 
+pub mod int;
```

Comment:
How is this compiled out when not in `cfg(hax)`?

Reply:
Postponing that to another PR, #639